python/setup.cfg

```diff
@@ -42,6 +42,10 @@ package_dir =
     = src
 packages = find:
 python_requires = >=3.7
+install_requires =
+    pytz
+    python-dateutil
```

Comment:
I don't think that either one of these libraries is needed. I just did a thorough review of what `datetime` supports and I was able to get everything working.

The main reason why we don't need these is that we are _very_ strict about date/time conversions and only support a limited set of strings in ISO 8601 formats, and support only offsets rather than zone strings.

Reply:
removed both `pytz` and `python-dateutil`